package.json

```diff
@@ -74,7 +74,7 @@
     "eslint-plugin-prettier": "2.3.1",
     "eslint-plugin-react": "7.3.0",
     "expect": "1.20.2",
-    "expect-jsx": "2.6.0",
+    "expect-jsx": "4.0.0",
```

Comment:
That's already one thing that we won't have to do using renovate :)

Reply:
Yep, but the warnings came from this package. So how we handle that? We use renovate or we can upgrade the dependency manually?

Reply:
That's actually a good thing to not have to do it with Renovate. That's probably a strategy that we should adopt more often 👍